mongodb/mongo_setup.py

```diff
@@ -0,0 +1,30 @@
+from mongoengine import connect, disconnect
+from pydantic_settings import BaseSettings
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+# Define MongoDB settings class
+class MongoDBSettings(BaseSettings):
```

Comment:
This (a class of app settings from .env) should not be here. Create a separate `env_utils.py` files to store and handle them.